soepy/python/shared/shared_auxiliary.py

```diff
@@ -140,18 +138,20 @@ def calculate_total_utilities(attr_dict, consumption_utilities, optim_paras):
 
 
 def calculate_continuation_values(
-    attr_dict, mapping_states_index, periods_emax, period, educ_years_idx, exp_p, exp_f
+    model_params,
+    mapping_states_index,
+    periods_emax,
+    period,
+    educ_years_idx,
+    exp_p,
+    exp_f,
 ):
     """Obtain continuation values for each of the choices."""
 
-    # Unpack attributes from the model specification:
-    num_choices = attr_dict["GENERAL"]["num_choices"]
-    num_periods = attr_dict["GENERAL"]["num_periods"]
-
     # Initialize container for continuation values
-    continuation_values = np.tile(MISSING_FLOAT, num_choices)
+    continuation_values = np.tile(MISSING_FLOAT, model_params.num_choices)
```

Comment:
Do we really need the MISSING_FLOAT variable, can't we just use np.nan everywhere? This might come from my use in respy, but there it is motivated to align the FORTRAN and PYTHON implementations and NP.NAN is not available in FORTRAN. If valid concern, please remove MISSING_INT and MISSING_FLOAT everywhere.

Reply:
I removed MISSING_FLOAT everywhere along with all artefacts that came with it. However, when replacing MISSING_INT with np.nan all tests crashed. I have therefore kept MISSING_INT for now.